internal/schema/fabric_common_schemas.go

```diff
@@ -0,0 +1,28 @@
+package schema
+
+import "github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+
+func readOrderSch() map[string]*schema.Schema {
```

Comment:
This shouldn't be needed. The correct order schema should be in `internal/fabric/schema/fabric_common_schemas.go`